src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.cpp

```diff
@@ -1072,6 +1090,11 @@ void ov::npuw::LLMInferRequest::infer() {
     OPENVINO_ASSERT(ov::element::i64 == attention_mask->get_element_type());
     OPENVINO_ASSERT(ov::element::i64 == position_ids->get_element_type());
 
+    // Eagle3: Accept and validate hidden state inputs
+    m_eagle3_ext.store_hidden_state_inputs(inputs, [this](const ov::Output<const ov::Node>& port) {
```

Comment:
As we have one input now, getter seems a bit redundant, lets stick to the `infer_request` and input ports map here